library/core/src/cell.rs

```diff
@@ -562,6 +562,16 @@ impl<T: Default> Cell<T> {
 #[unstable(feature = "coerce_unsized", issue = "27732")]
 impl<T: CoerceUnsized<U>, U> CoerceUnsized<Cell<U>> for Cell<T> {}
 
+// Allow types that wrap `Cell` to also implement `DispatchFromDyn`
+// and become object safe method receivers.
+// Note that currently `Cell` itself cannot be a method receiver
+// because it does not implement Deref.
+// In other words:
+// `self: Cell<&Self>` won't work
+// `self: CellWrapper<Self>` becomes possible
+#[unstable(feature = "dispatch_from_dyn", issue = "none")]
```

Comment:
This probably ought to point at an actual issue, rather than none.

Reply:
The trait itself currently doesn't have an issue associated with it.
https://github.com/rust-lang/rust/blob/bed4ad65bf7a1cef39e3d66b3670189581b3b073/library/core/src/ops/unsize.rs#L115-L119

We could associate the trait and it's usages with [arbitrary_self_types ](https://github.com/rust-lang/rust/issues/44874) but I'm not sure if that is desired or should be done in this PR.